@ndlib/gatsby-theme-marble/src/assets/i18n/json/text.default.en.json

```diff
@@ -6,9 +6,14 @@
   "noUserPage": "The user <1>{{ userName }}</1> does not exist or you are not authorized to view this page.",
   "userLogoutPage": "You have been successfully logged out.",
   "userPage": {
-    "loginPrompt": "<0>Log in</0> to access and edit your own content.",
+    "loginPrompt": "<0>Log in</0> to access and edit your own content."
     "followPrompt": "<0>Log in</0> to follow users or access and edit your own content."
```

Comment:
Missing comma after "loginPromt" line. Looks like this is making making unit test fail and should be breaking the build.